apis/capabilities/v1beta1/product_types.go

```diff
@@ -353,7 +353,12 @@ type OIDCSpec struct {
 	IssuerType string `json:"issuerType"`
 
 	// Issuer is the OIDC issuer
-	IssuerEndpoint string `json:"issuerEndpoint"`
+	// +optional
+	IssuerEndpoint string `json:"issuerEndpoint,omitempty"`
+
+	// IssuerEndpointRef  is the reference to OIDC issuer Secret that contains IssuerEndpoint
+	// +optional
+	IssuerEndpointRef corev1.SecretReference `json:"issuerEndpointRef,omitempty"`
```

Comment:
Does not apply for all cases, but usually optional fields should be pointers to differentiate from ZERO value from not set.

Reply:
done. Thank you